v2/lints/apple/lint_e_server_cert_valid_time_longer_than_398_days.go

```diff
@@ -29,7 +29,7 @@ func (l *serverCertValidityTooLong) Initialize() error {
 }
 
 func (l *serverCertValidityTooLong) CheckApplies(c *x509.Certificate) bool {
-	return util.IsServerAuthCert(c)
+	return util.IsServerAuthCert(c) && !c.IsCA
```

Comment:
@cardonator This should really have a unit test to prevent a regression.

Reply:
Good call, I will get a PR in for that.

Reply:
Thank you!